### PR TITLE
remove encoded option

### PR DESCRIPTION
**Pull request by oneilcin**

### CODE REVIEW

**Comment by joejulian** [comment]
I was suggesting we simplify further and just document that the `CLUSTER_PRIVATE_KEY`, `NODE_CLUSTER_PRIVATE_KEY`, `CLUSTER_PASSPHRASE` and `NODE_CLUSTER_PASSPHRASE` are required to be base64 encoded. Then we can take out all the OS specific stuff from the script.

**Comment by oneilcin** [comment]
Will take doing the base64 encoding out, as Joe suggested.  Add instructions to encode in the readme

File: README.md [hunk 1]
```diff
@@ -32,20 +32,15 @@ make
...
-- To specify a master and node ssh secret that is already encoded
+The private keys must be base64 encoded.
+- Define the base64 encoded ssh key for master and node environmentals
 ```bash
-export CLUSTER_PRIVATE_KEY_ENCODED=yourencodedsecret
-export CLUSTER_PASSPHRASE=yourpassphrase
-export NODE_CLUSTER_PRIVATE_KEY_ENCODED=yourencodedsecret
-export NODE_CLUSTER_PASSPHRASE=yourpassphrase
+export CLUSTER_PRIVATE_KEY=$(base64 ~/.ssh/id_rsa | tr -d \\r\\n)
+export NODE_CLUSTER_PRIVATE_KEY=$(base64 ~/.ssh/id_rsa | tr -d \\r\\n)
```
Comment (alejandroEsc) [inline README.md]: this makes the assumption the key we are using is our own, i dont think i agree with that, unless I am missing something?

File: clusterctl/examples/ssh/generate-yaml.sh [hunk 2]
```diff
@@ -61,53 +61,24 @@ fi
 
 # TODO Fill out the generation pieces as we need them.
 
-if [ -z ${CLUSTER_PRIVATE_KEY_PLAIN+x} ] && [ -z ${CLUSTER_PRIVATE_KEY_ENCODED+x} ]; then
-    echo "Please generate a valid Cluster Private Key and export the key file contents to CLUSTER_PRIVATE_KEY_PLAIN or CLUSTER_PRIVATE_KEY_ENCODED."
+if [ -z ${CLUSTER_PRIVATE_KEY+x} ]; then
+    echo "Please generate a valid base64 encoded cluster private key and export the key file contents to CLUSTER_PRIVATE_KEY."
     exit 1
```
Comment (alejandroEsc) [inline clusterctl/examples/ssh/generate-yaml.sh]: I wonder if there is a way to test if something is base64 encoded, that would be nice, not asking a change just asking if you knew?

File: README.md (hunk 1 above)
Comment (joejulian) [reply README.md]: It's an example.

File: clusterctl/examples/ssh/generate-yaml.sh (hunk 2 above)
Comment (joejulian) [reply clusterctl/examples/ssh/generate-yaml.sh]: Could do it with grep and regex, but that can come in a later PR.

**Pull request merged**